src/test/shared/cloudformation/templateRegistry.test.ts

```diff
@@ -100,15 +100,15 @@ describe('CloudFormation Template Registry', async () => {
         // other get cases are tested in the add section
         describe('getRegisteredTemplate', async () => {
             it('returns undefined if the registry has no registered templates', () => {
-                assert.strictEqual(testRegistry.getRegisteredTemplate('template.yaml'), undefined)
+                assert.strictEqual(testRegistry.getRegisteredTemplate('/template.yaml'), undefined)
```

Comment:
relative paths are no longer accepted in the templateRegistry

Reply:
Wasn't sure if this would work on Windows, but looks like it clears.